crates/oxc_ast/src/ast_builder.rs

```diff
@@ -879,6 +879,23 @@ impl<'a> AstBuilder<'a> {
         }))
     }
 
+    pub fn class_constructor(&self, span: Span, value: Box<'a, Function<'a>>) -> ClassElement<'a> {
+        ClassElement::MethodDefinition(self.alloc(MethodDefinition {
+            span,
+            key: self.property_key_expression(self.identifier_reference_expression(
+                IdentifierReference::new(SPAN, "constructor".into()),
```

Comment:
I'm not sure we can use `SPAN` here. If we can use `SPAN` we can add more useful helper functions, such as `require_call`

Reply:
I'm still unsure about empty spans and their implications, but we'll use it until problems come up.